components/core/src/os/users/windows.rs

```diff
@@ -12,26 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::env;
 use std::path::PathBuf;
 
+use habitat_win_users::account::*;
+
 extern "C" {
     pub fn GetUserTokenStatus() -> u32;
 }
 
-pub fn get_uid_by_name(owner: &str) -> Option<u32> {
-    unimplemented!();
+fn get_sid_by_name(name: &str) -> Option<String> {
```

Comment:
Hmm I'm getting compile errors with get_sid_by_name<T: AsString> and not finding refs to that on google. Is this a typo?